docs/admin/support.rst

```diff
@@ -5,7 +5,7 @@ Getting support for Weblate
 Weblate is copylefted libre software with community support.
 Subscribers receive priority support at no extra charge. Prepaid help packages are
 available for everyone. You can find more info about current support
-offerings at <https://weblate.org/support/>.
+offerings at `https://weblate.org/support <https://weblate.org/support/>`_.
```

Comment:
What is point in duplicating links? See my comments on https://github.com/WeblateOrg/weblate/pull/5742

Reply:
They currently look like 
![bilde](https://user-images.githubusercontent.com/13802408/112740211-a0157c00-8f6a-11eb-97bd-808b43c5c563.png)
Not sure if the URL only makes it a hotlink, but marking it up makes it easy to change the link later on if needed.
The last / makes it easy to see what part is the actual link, and which part is the displayed name.

I'll change it to URLs only.

Reply:
corrected, changed to URLs only